php/libraries/BVL_Feedback_Panel.class.inc

```diff
@@ -62,16 +62,21 @@ class BVL_Feedback_Panel
      */
     function display()
     {
-        $this->tpl_data['candID']
-            = $this->_feedbackThread->_feedbackCandidateProfileInfo["CandID"];
-        $candidateID     = $this->_feedbackThread
-            ->_feedbackCandidateProfileInfo["CandID"];
-        $candidateObject = Candidate::singleton($candidateID);
-        $this->tpl_data['pscid']          = $candidateObject->getPSCID();
-        $this->tpl_data['sessionID']      = $this->_feedbackThread
-            ->_feedbackCandidateProfileInfo["SessionID"];
-        $this->tpl_data['commentID']      = $this->_feedbackThread
-            ->_feedbackObjectInfo["CommentID"];
+        $feedbackProfile = $this->_feedbackThread->_feedbackCandidateProfileInfo;
+        $candidateID     = $feedbackProfile["CandID"];
+
+        $this->tpl_data['candID'] = $candidateID;
+
+        $candidateObject         = Candidate::singleton($candidateID);
+        $this->tpl_data['pscid'] = $candidateObject->getPSCID();
+
+        $this->tpl_data['sessionID'] = isset($feedbackProfile["SessionID"]) ?
```

Comment:
Weird. I don't remember writing any of this, lol. But Github says it's my pull request. I guess it is...